spec/support/webmock_helper.rb

```diff
@@ -1,16 +1,20 @@
 def stubbed_requests
-  stub_request(:post, "https://api.twilio.com/2010-04-01/Accounts/articuno34/Messages.json")
+  # Short IO
+  stub_request(:post, "https://api.short.io/links")
     .with(
-      body: {From: "+15555555555", Body: "Execute Order 66 - https://42ni.short.gy/jzTwdF", To: "+12222222222"},
+      body: {originalURL: "https://www.youtube.com/watch?v=dQw4w9WgXcQ", domain: "42ni.short.gy"}.to_json,
       headers: {
-        "Content-Type" => "application/x-www-form-urlencoded",
-        "Authorization" => "Basic QWxhZGRpbjpvcGVuIHNlc2FtZQ=="
+        "Accept" => "application/json",
+        "Accept-Encoding" => "gzip;q=1.0,deflate;q=0.6,identity;q=0.3",
+        "Authorization" => "1337",
+        "Content-Type" => "application/json",
+        "User-Agent" => "Ruby"
       }
     )
-    .to_return(body: "{\"error_code\":null, \"status\":\"sent\", \"body\":\"Execute Order 66 - https://42ni.short.gy/jzTwdF\"}")
+    .to_return(status: 200, body: "{\"shortURL\":\"https://42ni.short.gy/jzTwdF\"}", headers: {})
   stub_request(:post, "https://api.short.io/links")
     .with(
-      body: {originalURL: "https://www.youtube.com/watch?v=dQw4w9WgXcQ", domain: "42ni.short.gy"}.to_json,
+      body: "{\"originalURL\":\"https://casavolunteertracking.org/case_contacts/new\",\"domain\":\"42ni.short.gy\"}",
```

Comment:
do not use static urls, they break in other environments

Reply:
See above comment for fix